apps/dot_ext/views/authorization.py

```diff
@@ -84,9 +106,16 @@ def form_valid(self, form):
                 application=application,
                 share_demographic_scopes=share_demographic_scopes,
                 scopes=scopes,
-                allow=allow)
+                allow=allow,
+                access_token_delete_cnt=access_token_delete_cnt,
+                refresh_token_delete_cnt=refresh_token_delete_cnt)
             return response
 
+        # Did the beneficiary choose not to share demographic scopes, or the application does not require them?
+        if share_demographic_scopes == "False" or (allow is True and application.require_demographic_scopes is False):
```

Comment:
I think this is probably ok, but when looking at the SimpleAllowForm class definition I'm a little confused.  There is logic in that form clean method that converts share_demographic_scopes to a boolean, but the actual return on that value is still a string?

https://github.com/CMSgov/bluebutton-web-server/blob/9179c6e9678a15ccf3b1b10711c031c1488d8ffd/apps/dot_ext/forms.py#L173-L177

Reply:
@njdister Oops. Was some code I forgot to change back when trying to get the boolean form field working right. I updated the code to clean that up. TY!  The return value will be a string.

Reply:
Ok great, final question: since you call lower() on `share_demographic_scopes` comparison during form cleaning, should you also do it here for safety?

Reply:
@njdister I removed that `.lower()` since it is not needed. The template is always going to post "False" or "True" for that radio button value.